.github/workflows/failure_free_docs.yml

```diff
@@ -0,0 +1,29 @@
+name: failure_free_docs
+on:
+  pull_request:
+    paths:
+      - "docs/fail/**"
```

Comment:
Unfortunately when the `on: ... paths` option is used it causes the job to not appear in the list at all if it's skipped, which means the job can't be included in the required status checks list:
https://github.com/orgs/community/discussions/54877

This means this check wouldn't block the update PRs, and someone could accidentally merge them anyway, if they didn't see the red result in the status checks.

As such, I think we'll need this workflow to always run instead - a bit like the check-changelog workflow always runs.